ExperimentIntegrationTests/xcrun.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import subprocess
+
+logging.getLogger(__name__).addHandler(logging.NullHandler())
+
+
+class XCRun(object):
```

Comment:
could we use the existing xcrun.py file in the SyncIntegration tests? if so, we could move it to a helper or tooling folder...